gtsam/hybrid/tests/TinyHybridExample.h

```diff
@@ -43,12 +43,13 @@ inline HybridBayesNet createHybridBayesNet(size_t num_measurements = 1,
   // Create Gaussian mixture z_i = x0 + noise for each measurement.
   for (size_t i = 0; i < num_measurements; i++) {
     const auto mode_i = manyModes ? DiscreteKey{M(i), 2} : mode;
+    DiscreteKeys modes{mode_i};
+    std::vector<GaussianConditional::shared_ptr> conditionals{
+        GaussianConditional::sharedMeanAndStddev(Z(i), I_1x1, X(0), Z_1x1, 0.5),
+        GaussianConditional::sharedMeanAndStddev(Z(i), I_1x1, X(0), Z_1x1, 3)};
     bayesNet.emplace_shared<HybridGaussianConditional>(
         KeyVector{Z(i)}, KeyVector{X(0)}, DiscreteKeys{mode_i},
-        std::vector{GaussianConditional::sharedMeanAndStddev(Z(i), I_1x1, X(0),
-                                                             Z_1x1, 0.5),
-                    GaussianConditional::sharedMeanAndStddev(Z(i), I_1x1, X(0),
-                                                             Z_1x1, 3)});
+        HybridGaussianConditional::Conditionals(modes, conditionals));
```

Comment:
So, this is the case of a single key, so here you can use a vector, as we discussed. You would then specify a single DiscreteKey, not a set.

Reply:
Ah sorry about that, I had already changed to the DecisionTree constructor before we discussed the single key option. Updated.

Reply:
FYI this is for `HybridGaussianConditional` which can have multiple discrete parents. I've kept this for multiple discrete keys mainly for the wrapper.
Should I still go ahead and update this?

Reply:
I'm going ahead and doing this in the interest of time. If a use case comes up where we need multiple discrete parents + the vector of conditionals, we can re-evaluate then. :)